kele.el

```diff
@@ -1252,15 +1325,17 @@ Defaults to the currently active context as set in
                   :version version
                   :namespace namespace
                   :context context)))
-    :if (lambda ()
-          (-let (((&alist 'group-version gv
-                         'kind kind
-                         'context context)
-                 (oref transient--prefix scope)))
-            (kele--resource-has-verb-p
-             kele--global-discovery-cache
-             gv kind "get"
-             :context context)))
+    ;; TODO: Not super sure how this plays w/ a group-version that could be
+    ;; always updating via infix
```

Comment:
Be sure to reopen #81. This regression is fine since switching on group-version is arguably more valuable.